packages/devtools-network-console/public/index.html

```diff
@@ -13,7 +13,7 @@
         default-src 'self' blob:;
         style-src 'self' 'unsafe-inline' https://devtools.azureedge.net;
         font-src 'self' https://static2.sharepointonline.com https://spoprod-a.akamaihd.net https://devtools.azureedge.net;
-        script-src 'self' 'unsafe-inline' https://devtools.azureedge.net;
+        script-src 'self' 'unsafe-inline' 'unsafe-eval' https://devtools.azureedge.net;
```

Comment:
This appears to be a new issue in the browser. It should be removed later, but needs to be here for now, or else it won't run.